SimpleBrowser/SimpleBrowser.nuspec

```diff
@@ -17,5 +21,6 @@
   <files>
     <file src="SimpleBrowser\bin\Release\net40\SimpleBrowser.dll" target="lib\net40\SimpleBrowser.dll" />
     <file src="SimpleBrowser\bin\Release\net40\RazorHosting.dll" target="lib\net40\RazorHosting.dll" />
+    <file src="SimpleBrowser\bin\Release\netstandard2.0\SimpleBrowser.dll" target="lib\netstandard2.0\SimpleBrowser.dll" />
```

Comment:
@kevingy 
We should consider also including the xml documentation here in the package. That way the documentation is also (partially) visible in Visual Studio (IntelliSense, Object Browser, ...).

Reply:
@Styxxy I agree. Good idea!